README.md

```diff
@@ -125,6 +146,31 @@ Unstable options are not available on stable/beta with Rustfmt v1.x.
 
 See the configuration documentation on the Rustfmt [GitHub page](https://rust-lang.github.io/rustfmt/) for details (look for the `unstable_features` section).
 
+### Differences in rustfmt versions
+
+Aside differences in [configuration options](#configuring-rustfmt), versions of rustfmt may differ
+in formatting behavior.
+
+#### Default formatting of submodules
+
+On an invocation `rustfmt lib.rs`, rustfmt 1.x would format both "lib.rs" and any out-of-file
+submodules referenced in "lib.rs".
```

Comment:
Maybe worth mentioning 1.x's `skip_children` here. Thoughts?
```suggestion
submodules referenced in "lib.rs". To prevent formatting out-of-file modules, `skip_children` must be used and set to `true`.
```

Reply:
done.